### PR TITLE
Fix stokes flow.

### PR DESCRIPTION
**Pull request by termi-official**

### CODE REVIEW

File: docs/src/literate/stokes-flow.jl [hunk 1]
```diff
@@ -461,7 +462,7 @@ function main()
     vtk_grid("stokes-flow", grid) do vtk
         vtk_point_data(vtk, dh, u)
     end
-    Sys.isapple() || @test norm(u) ≈ 0.32254330524111213 #src
+    Sys.isapple() || @test norm(u) ≈ 0.322804206709586 #src
```
Comment (fredrikekre) [inline docs/src/literate/stokes-flow.jl:465]: Good catch. I wonder if this fixes the example on macOS:
```suggestion
    @test norm(u) ≈ 0.322804206709586 #src
```

Comment (termi-official) [reply docs/src/literate/stokes-flow.jl]: Btw, do you know under which circumstances the mean value constraint is valid? It seems like we require no slip in a Stokes problem to make it work, but I cannot find material on the theory.

**Comment by termi-official** [comment]
Is it valid to have periodic velocity but non-periodic pressure?

**Comment by termi-official** [comment]
I think gmsh generates a slightly different mesh on OSX than on windows and linux, which caused CI to fail.

**Comment by fredrikekre** [comment]
> I think gmsh generates a slightly different mesh on OSX than on windows and linux, which caused CI to fail.

Ah, that would definitely explain the small changes in the result, never thought of that as an option.

I made another bugfix to this example: https://github.com/Ferrite-FEM/Ferrite.jl/commit/d9446c2bfae305c859fce071000bcd1056887e48 but with the previous definition of `constrained_dof` that was actually correct before :)

**Pull request merged**